src/model/entities/part.ts

```diff
@@ -31,10 +34,13 @@ export interface PartInterface {
 
   timings?: PartTimings
   endState?: PartEndState
+
+  defaultPart?: IngestedPart
```

Comment:
I'm not quite sure i understand, what is the responsibility of the `defaultPart`?

Reply:
Maybe the naming isn't optimal.

It's the "ingested/default" Part. It's the values to fall back on when resetting the Part.
So we can change the values of the Part to our hearts desire. Then when we reset the Part, we change all the values back to the values from the `defaultPart`.

We are changing the values back on the Part instead of creating a new Part object instance so we can keep the same object references that is being used by Rundown and Segment.
Also, by letting the Part have its own default Part, we don't need to worry about mappers between `IngestedPart` and `Part` which is something we don't have access to within our entities.